contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,354 @@
+
+1. Intros through your own network, people you worked with, went to school with, where your family work or went to school. Everybody has _some_ network – don't feel like you need to be a sales legend to do this.
+2. If you're in an accelerator, message other companies. Be personalized.
+3. People from companies that your product is perfect for.
+4. Events. You may be able to get free tickets via a partner.
+
+In general, do not do marketing (paid ads / content) expecting to get a sale – these are more scalable methods. Be hands on. Other companies in your space will have heavily invested in how to do outbound or marketing, so the most likely way you can win is on quality and personalization with your approach.
+
+Keep yourself accountable – set some goals for meetings booked and stick to them.
+
+As with booking feedback calls, be hyper responsive. If you respond to people within minutes (whether booking a meeting, or progressing a deal), you'll get them while they're available and happy to engage. I used to obsessively respond to people on LinkedIn all day while in this phase.
+
+When it comes to pricing, reduce the risk for your earliest customers. Charge them monthly rather than annually, unless they prefer the latter.
+
+There are multiple methods available. You can charge a recurring monthly rate, or you can charge on usage.
+
+Once you're on a call, get the prospect to open up about their challenges, and figure out with them if your product could solve their problems. It's worth qualifying the lead as you do this. The simplest method is BANT (Budget, Authority, Need, Timeline).
+
+Layer your questions.
+
+* Level 1 - "What are the requirements for this project?", "how will you pick a vendor?", "do you have a set budget?"
+* Level 2 - "This sounds important, why?", "would you consider a product that didn't include that feature?" and so on. 
+* Level 3 - "If your end users were thoroughly trained and using this product effecitvely, what impact would that have on your revenue?", "what does solving this mean for you?".
+
+Prioritize time with those that are best-qualified.
+
+Once someone decides to move forward, you'll need something for people to sign. YC provide a [free sales agreement](https://www.ycombinator.com/sales_agreement/) for SaaS companies. Be flexible around your earliest customers for the terms they need – it's not about maximizing revenue or contract length, it's about getting your first deal signed and learning.
+
+## The why, when, and how of pivoting
+
+A lot of the hustle/grind culture around startups would make you think that the very best startups took years and years of work to get off the ground. The reality, once we got into YC, was the opposite. It felt like the _majority_ of the best companies there pivoted – most ideas people were working on needed to change. Since most startups fail, you're probably doing something wrong if you _don't_ pivot. At PostHog, we pivoted six times in the first six months!
```

Comment:
```suggestion
A lot of the hustle/grind culture around startups would have you believe the very best startups took years and years of work to get off the ground. The reality, once we got into YC, was the opposite. 

It felt like the _majority_ of the best companies there pivoted. Since most startups fail, you're probably doing something wrong if you _don't_ pivot! At PostHog, we pivoted six times in the first six months!
```